coalib/processes/Processing.py

```diff
@@ -751,7 +755,12 @@ def execute_section(section,
                              results (bear names are key) as well as the
                              file dictionary.
     """
-    if debug:
+    debug_bears = (False
+                   if 'debug_bears' not in section or (
+                       section['debug_bears'].value == 'False') else (
+                            typed_list(str)(section['debug_bears'])))
```

Comment:
Indentation: 8 spaces back

Reply:
Every time I am trying to make the `8 spaces back`  `PEP8Bear` made the change during running coala  :sweat_smile: